### PR TITLE
feat: [SSCA-1091]: Add Remediation Tracker route

### PR DESCRIPTION
**Pull request by deepesh-ui**
#### Screenshots

Screenshot 2023-12-12 at 9 03 58 AM

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
- MessageMetadata: `trigger messagecheck`
- GitLeaks: `trigger gitLeaks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

### CODE REVIEW

**Comment by deepesh-ui** [comment]
trigger messagecheck

File: src/modules/10-common/RouteDefinitionsV2.ts [hunk 1]
```diff
@@ -974,6 +974,7 @@ const routes = {
   // ssca module routes
   toSSCA: withModeModuleAndScopePrefix(() => ''),
   toSSCAArtifacts: withModeModuleAndScopePrefix(() => '/artifacts'),
+  toRemediationTrackers: withModeModuleAndScopePrefix(() => '/remediation-trackers'),
```
Comment (udayvunnam) [inline src/modules/10-common/RouteDefinitionsV2.ts:977]: nit: toRemediationTracker

File: src/modules/75-ssca/components/SSCASideNavLinks.tsx [hunk 2]
```diff
@@ -44,6 +57,7 @@ const SSCASideNavLinks = (mode: NAV_MODE): React.ReactElement => {
             to={routes.toPipelines({ accountId, projectIdentifier, orgIdentifier, module })}
             hidden={mode === NAV_MODE.ALL}
           />
+          {RemediationTrackerLink}
```
Comment (udayvunnam) [inline src/modules/75-ssca/components/SSCASideNavLinks.tsx:60]: We can use the SideNav.Link, hidden property.

```
<SideNav.Link
            icon="nav-pipeline"
            label={getString('pipelines')}
            to={routes.toPipelines({ accountId, projectIdentifier, orgIdentifier, module })}
            hidden={mode === NAV_MODE.ALL}
          />
```

**Comment by udayvunnam** [comment]
Please add this for old nav as well. We need to support both until new nav GA

**Review by udayvunnam — APPROVED** [review]

File: src/modules/10-common/RouteDefinitions.ts [hunk 3]
```diff
@@ -2418,6 +2418,7 @@ const routes = {
   toSSCA: withAccountId(() => '/ssca'),
   toSSCAOverview: withAccountId(() => '/ssca/overview'),
   toSSCAArtifacts: withProjectLevelRoute(() => '/artifacts'),
+  toRemediationTracker: withProjectLevelRoute(() => '/remediation-trackers'),
```
[the remark this answers is not in the document]
Comment (deepesh-ui) [reply src/modules/10-common/RouteDefinitions.ts:2421]: it a list so kept it as `remediation-trackers`

**Review by bhavya-sinha — APPROVED** [review]

**Pull request merged**